blog/2025-07-21-openhab-5-0-release.md

```diff
@@ -0,0 +1,416 @@
+
+- The number of **_Point+Property_** tag default values increased from under 3% to 40% of all bindings' channel types (or 81% of non-advanced channel types).
+- The number of **_Equipment_** tag default values increased from zero to 74% of all bindings' thing types.
+- Some bindings now apply **_Equipment_** and **_Point+Property_** tag default values dynamically depending on the actual attributes of Things and Channels in the Inbox. This further increases the tag coverage percentages above.
+
+## Main UI Enhancements
+
+_Florian Hotze ([@florian-h05](https://github.com/florian-h05)), openHAB Maintainer_
+
+### Sitemap Editor Drag & Drop
+
+<img src="/uploads/2025-07-21-openhab-5-0-release/sitemap-edit-dragdrop.gif" class="img-beside-text-right" alt="Sitemap Edit: Drag & Drop" />
+
+When designing sitemaps through the UI, it was very cumbersome to move elements around, as you had to delete and re-add them in the desired order, or cut and paste the DSL code in the code tab.
+With openHAB 5.0, we have introduced a drag and drop feature to the sitemap editor, allowing you to easily rearrange elements in the sitemap:
+
+Many thanks to Mark Herwege ([@mherwege](https://github.com/mherwege)) for implementing this feature!
+<div style="clear:both;"></div>
+
+### Model Editor Drag & Drop
+
+<div class="img-beside-text-right" style="display: flex; justify-content: space-between;">
+  <img style="width: calc(50% - 3px);" src="/uploads/2025-07-21-openhab-5-0-release/model-dragdrop-1.gif"/>
+  <img style="width: calc(50% - 3px);" src="/uploads/2025-07-21-openhab-5-0-release/model-dragdrop-2.gif"/>
+</div>
+
+Similarly to the sitemap editor, the model editor now supports drag and drop for moving semantic tags around.
+This not only makes it faster to build your semantic model, it also makes it easier to ensure that your model is valid and follows the best practices by introducing numerous validation checks.
+
+A big thank you to Mark Herwege ([@mherwege](https://github.com/mherwege)) for working on this game-changing feature!
+<div style="clear: both;"></div>
+
+### Regenerate Rules from Templates
+
+_Ravi Nadahar ([@Nadahar](https://)), openHAB Contributor_
+
+Previously, rules created from rule templates were static and could not be updated if the template was changed.
+It was necessary to delete and recreate the rules from the template to apply any updates or changes to the parameters used in the template.
+openHAB 5.0 introduces a new feature that allows you to regenerate rules from templates, making it easier to keep your rules up to date with the latest changes in the templates.
+This also allows you to change the parameters used in the template without having to delete and recreate the rules.
+
+You can find more details in [the documentation](/docs/tutorial/rules_basic.html#updates-to-templates).
```

Comment:
```suggestion
You can find more details in [the documentation](/docs/tutorial/rules_basic.html#update-a-rule).
```
Ideally, the link should be this instead. Sorry for not doing it right the first time, but it's not "wrong" the way it is either.